### ISSUE TITLE
Favicons should have transparent background

### ISSUE THREAD

**Issue by spMohanty** [AICrowd/AIcrowd#831]

**Comment by nemaniarjun** [comment]
@jsnrynlds Do we have the favicons somewhere?

**Comment by jsnrynlds** [comment]
@nemaniarjun Yes, they're at: https://github.com/AIcrowd/AIcrowd-Design/tree/master/assets/img/favicons 

Should already have transparent backgrounds but have included a PNG (avatar.png) you can use for any additional ones you may need.

**Comment by nemaniarjun** [comment]
@jsnrynlds anyway we can make custom favicons for the 3 components (Rails can use default) for gitlab and discourse? 

For example here is one I made for storybook: https://ui-storybook.aicrowd.com/?path=/story/*

**Comment by jsnrynlds** [comment]
@nemaniarjun I think we should just use the standard one – it's not great practice to start tweaking the logomark.

**Comment by nemaniarjun** [comment]
Cool!

**Comment by nemaniarjun** [comment]
@spMohanty is there any other components apart from 
1. Rails : Done!
2. Discourse: Doesnt use the transparent one!
3. Gitlab : Not Done!

**Comment by nemaniarjun** [comment]
Looks to be done on discourse. @spMohanty @skbly7 do we change this for gitlab as well? Otherwise we can close this issue.

**Comment by nemaniarjun** [comment]
@skbly7 bump

**Comment by skbly7** [comment]
Updated the favicon on Gitlab as well. Although we need a new version of favicon which is transparent BUT have white color under eyes & mouth, so they look prettier.

Discourse & GitLab needed png, and we had one so it's done there. Rails will need an update.
Example: https://discourse.aicrowd.com/uploads/default/optimized/1X/078b972715bf4f61f853c83ed6bea8afac59fa89_2_32x32.png

@jsnrynlds what do you think about it? We are also trying to use white background logo in other brandings as well. Due to which this folder need an update: https://github.com/AIcrowd/AIcrowd-Design/tree/master/assets/img/favicons

**Comment by jsnrynlds** [comment]
Hey @skbly7, I think using a white-background behind the eyes and mouth is a good idea. I'll update the favicon in the design repo soon.

**Comment by jsnrynlds** [comment]
Updated favicon in design repo.

**Comment by skbly7** [comment]
Thanks a lot, @jsnrynlds. 
@nemaniarjun you can take it forward for applying to the website here.
